corehq/apps/app_manager/management/commands/flag_domains_using_search_filter.py

```diff
@@ -0,0 +1,40 @@
+from django.core.management import BaseCommand
+
+from dimagi.utils.chunked import chunked
+from corehq import toggles
+from corehq.apps.app_manager.dbaccessors import domain_has_apps, get_app_ids_in_domain, get_current_app
+from corehq.util.log import with_progress_bar
+
+
+class Command(BaseCommand):
+    help = "One-time command to enable search filter flag where 'search filter' is already in use"
+
+    def handle(self, **options):
+        flag_domains_using_search_filter()
+
+
+def flag_domains_using_search_filter():
+    domains = toggles.USH_CASE_CLAIM_UPDATES.get_enabled_domains()
+    for chunk in chunked(with_progress_bar(domains), 100):
+        for domain in chunk:
+            enabled = domain_has_apps(domain) and _any_app_uses_search_filter(domain)
+            toggles.USH_SEARCH_FILTER.set(domain, enabled, namespace=toggles.NAMESPACE_DOMAIN)
+
+
+def _any_app_uses_search_filter(domain):
+    for app_id in get_app_ids_in_domain(domain):
+        try:
+            app = get_current_app(domain, app_id)
+        except Exception:
```

Comment:
I think it is probably fine to let the command continue (as you mentioned before), but it would be good to log the apps that failed so that we can follow up with them later. Broken apps in production are not good, even if we don't have to really worry about them in staging.

Reply:
How likely is it that an error encountered here would already be getting recorded somewhere? I'm not really sure where or what level of detail would be most useful (if it even catches anything).

Reply:
@nospame great question-- I think logging the domain and exception thrown would be helpful for followup. It would be hard to predict whether this is recorded elsewhere for every instance of error right now, but then a followup PR adding in additional logging could happen.